docs/advanced-topics/hadoop.md

```diff
@@ -72,68 +72,37 @@ by leveraging Apache Spark.
 
 ### OLAP Traversals with Spark Local
 
-The backend demonstrated here is Cassandra for the OLAP example below.
-Additional configuration will be needed that is specific to that storage
-backend. The configuration is specified by the
+OLAP Example below are showing configuration examples for default support 
+backends by JanusGraph. Additional configuration will be needed that 
+is specific to that storage backend. The configuration is specified by the
 `gremlin.hadoop.graphReader` property which specifies the class to read
 data from the storage backend.
 
-JanusGraph currently supports following graphReader classes:
+JanusGraph directly supports following graphReader classes:
 
 * `CqlInputFormat` for use with Cassandra 3
 * `CassandraInputFormat` for use with Cassandra 2
 * `HBaseInputFormat` and `HBaseSnapshotInputFormat` for use with HBase
 
-The following properties file can be used to connect a JanusGraph
-instance in Cassandra such that it can be used with HadoopGraph to run
-OLAP queries.
-
-```conf
-# read-cassandra-3.properties
-#
-# Hadoop Graph Configuration
-#
-gremlin.graph=org.apache.tinkerpop.gremlin.hadoop.structure.HadoopGraph
-gremlin.hadoop.graphReader=org.janusgraph.hadoop.formats.cql.CqlInputFormat
-gremlin.hadoop.graphWriter=org.apache.tinkerpop.gremlin.hadoop.structure.io.gryo.GryoOutputFormat
-
-gremlin.hadoop.jarsInDistributedCache=true
-gremlin.hadoop.inputLocation=none
-gremlin.hadoop.outputLocation=output
-gremlin.spark.persistContext=true
-
-#
-# JanusGraph Cassandra InputFormat configuration
-#
-# These properties defines the connection properties which were used while write data to JanusGraph.
-janusgraphmr.ioformat.conf.storage.backend=cql
-# This specifies the hostname & port for Cassandra data store.
-janusgraphmr.ioformat.conf.storage.hostname=127.0.0.1
-janusgraphmr.ioformat.conf.storage.port=9042
-# This specifies the keyspace where data is stored.
-janusgraphmr.ioformat.conf.storage.cql.keyspace=janusgraph
-# This defines the indexing backend configuration used while writing data to JanusGraph.
-janusgraphmr.ioformat.conf.index.search.backend=elasticsearch
-janusgraphmr.ioformat.conf.index.search.hostname=127.0.0.1
-# Use the appropriate properties for the backend when using a different storage backend (HBase) or indexing backend (Solr).
-
-#
-# Apache Cassandra InputFormat configuration
-#
-cassandra.input.partitioner.class=org.apache.cassandra.dht.Murmur3Partitioner
-
-#
-# SparkGraphComputer Configuration
-#
-spark.master=local[*]
-spark.executor.memory=1g
-spark.serializer=org.apache.spark.serializer.KryoSerializer
-spark.kryo.registrator=org.apache.tinkerpop.gremlin.spark.structure.io.gryo.GryoRegistrator
+The following properties files can be used to connect a JanusGraph
```

Comment:
Again, I am not sure about that but I think we should use `property files` instead of `properties files` here.